resources/assets/sass/app.scss

```diff
@@ -8,7 +8,7 @@
 @import url('https://fonts.googleapis.com/css?family=Lato:700');
 
 // Font awesome
-@import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css");
```

Comment:
- Remove this causes the fontawesome icon missing. As i mentioned in the call last time, you have import the script in the layout blade to ensure the icon is showing